.eslintrc.js

```diff
@@ -93,7 +93,6 @@ module.exports = {
       parser: 'yaml-eslint-parser',
       rules: {
         'yml/sort-keys': 'error',
-        'yml/key-name-casing': ['error', { camelCase: true }],
```

Comment:
I'm feeling a little bit anxious about this. I like that we require camel-case and the only situation where I think it's reasonable to deviate from that pattern is when API fields are used directly in localization, in which case there can be an exception in those (rare) files with clear intent.